packages/sha256-browser/src/ie11Sha256.ts

```diff
@@ -1,34 +1,25 @@
 import { isEmptyData } from "./isEmptyData";
 import { SHA_256_HMAC_ALGO } from "./constants";
-import { Hash, SourceData } from "@aws-sdk/types";
+import { Checksum, SourceData } from "@aws-sdk/types";
 import { fromUtf8 } from "@aws-sdk/util-utf8-browser";
 import { CryptoOperation, Key, MsWindow } from "@aws-crypto/ie11-detection";
 import { locateWindow } from "@aws-sdk/util-locate-window";
 
-export class Sha256 implements Hash {
-  private operation: Promise<CryptoOperation>;
+export class Sha256 implements Checksum {
+  private readonly secret?: SourceData;
+  private operation?: Promise<CryptoOperation>;
```

Comment:
Suggestion: Same as before, use Definite Assignment Operator to tell TypeScript this will definitely be assigned. 
```suggestion
  private operation!: Promise<CryptoOperation>;
```